git.md

```diff
@@ -1,31 +1,70 @@
-### Development 
+### <a name="development"></a>Development
 
 1. Create a new branch from **base branch** (`master`) and name it  `<ticket id>-<your branch name>`
-1. Do your development and write `CHANGELOG.md` entry 
-1. Create Pull Request 
-   * Describe your changes. Use magic tags like `fixes <ticket id>` 
-   * If your work is in progress name PR `[WIP] <description>`
-   * If your work is done name it `[<ticket_id>]  <description>`
+1. Do your development and write `CHANGELOG.md` entry
+1. Create Pull Request
+   * Describe your changes. Use magic tags like `fixes <ticket id>`
+   * If your work is in progress create
+     [draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
+   * If your work is done convert draft PR to ready to review PR
    * Wait for 2 approvals in review (1 dev and product owner)
-1. Merge changes to the **base branch** 
+1. Merge changes to the **base branch**
 
-### Release 
+### <a name="release"></a>Release
 
-1. When changes in the **base branch** (`master`) are ready to be released _@wojt_ creates branch version `<a>.<b>.x` originating form the **base branch**
-1. _@wojt_ takes care of `CHANGELOG.md` entry in this branch (gives _Unreleased_ section appropriate version number)
-1. _@wojt_ creates Pull Request to `production` branch 
-   * we treat deployment of this branch as a test deployment for the release candidate 
-1. Release is performed by _@wojt_
-   * An appropriate `<a>.<b>.<c>` tag is added in `<a>.<b>.x` branch 
+1. When changes in the **base branch** (`master`) are ready to be released _@wojt_
+   creates branch version `<a>.<b>.x` originating form the `master` branch. This
+   branch is called **release candidate branch**.
+1. Into the **base branch** branch next development features can be merged
+   (these changes will not be release in described release).
+1. **release candidate branch** is automatically build and can be used to test release candidate
+1. When some problems are discovered in **release candidate branch** fixes are
+   done similar like described in [Fixes Development](#fixes-development) section.
+1. When everyting is OK, _@wojt_ runs script/github action which is performing following steps:
+   * **Version** is calculated using following algorithm:
+     * Find all tags with following pattern `<a>.<b>.<anything>`
+     * If no maching branch is found version is set to `<a>.<b>.0`
+     * If maching branches are found version is set to `<a>.<b>.<MAX(anything)>`
```

Comment:
MAX+1
If I understand correctly we are fixes are handled by this algorithm not only Development. Maybe Fixes section is not needed

Reply:
**base brach** is different than in normal development - this is the only modification described now in `Fixes` section.